easybuild/easyblocks/g/gromacs.py

```diff
@@ -51,6 +51,7 @@
 from easybuild.tools.run import run_cmd
 from easybuild.tools.toolchain.compiler import OPTARCH_GENERIC
 from easybuild.tools.systemtools import X86_64, get_cpu_architecture, get_shared_lib_ext, get_cpu_features
+from easybuild.tools.version import VERBOSE_VERSION
```

Comment:
Maybe it's better to rename this on import, to avoid confusion between the GROMACS version and the EasyBuild version:

```python
from easybuild.tools.version import VERBOSE_VERSION as EASYBUILD_VERSION

Reply:
+1

Reply:
Done